applications/asset_tracker_v2/src/modules/gps_module.c

```diff
@@ -52,6 +52,15 @@ static enum sub_state_type {
 /* GPS device. Used to identify the GPS driver in the sensor API. */
 static const struct device *gps_dev;
 
+static struct module_helpers {
+	/* Uptime set when GPS search is started. Used to calculate search time for
+	 * GPS fix and timeout.
+	 */
+	uint64_t start_uptime;
+	/* Numer of satellites tracked for a GPS fix or timeout. */
+	uint8_t satellites_tracked;
+} helpers;
```

Comment:
Why the name "helpers"? Shouldn't it rather be "stats" or "statistics"?

Reply:
No opinion here really. I can rename it to `stats`